mobile/mobile.go

```diff
@@ -96,13 +93,50 @@ type Mobile struct {
 }
 
 // RepoPath returns the actual location of the configured repo
-func (conf InitConfig) RepoPath() string {
-	return path.Join(conf.BaseRepoPath, conf.Seed)
+func (conf InitConfig) RepoPath() (string, error) {
```

Comment:
`RepoPath()` and `RepoExists()` are a little less nice to use if we scope the repo path with account address instead of seed. This is because we have to convert the provided seed to an account, which can fail, so we need to bubble that error up.